docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+`resourced` supports several swapping scenarios to provide more RAM to applications running in the foreground. To support these scenarios, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: Adds swap management features to memory resource controller. Depends on `CONFIG_MEMCG` and `CONFIG_SWAP`.
```

Comment:
Question : `CONFIG_MEMCG_SWAP` and `CONFIG_ZRAM` are described above, on line 97, as essential features that a user must enable to use `resourced` functionalities. 

In this added description, `CONFIG_MEMCG_SWAP` and `CONFIG_ZRAM` are guided again to be **enabled** to use swapping scenarios of `resourced`. 

Aren't they already enabled to use `resourced` according to line 97? I thought they are always enabled to use `resourced`, but it looks like a user can enable or not while using `resourced`.

Reply:
@mijin85cho, Thank you for pointing this out. I agree with you.

@steelman, please clarify.

Reply:
@steelman Would you also please check this comment?

Reply:
You are right, it isn't only the swap resourced module that requires these options. I merged the two lists to avoid confusion.